lmdeploy/serve/turbomind/chatbot.py

```diff
@@ -634,7 +645,7 @@ def stream_consumer(postprocess, res_queue, session, n_input_token,
                                         )]
                 last_token_id = None if output_ids.shape[
                     -1] == 0 else output_ids[-1, -1, -1]
-                if last_token_id == eos_id:
+                if last_token_id in filters:
```

Comment:
`session.sequence_length` is altered only when `last_token_id` is `eos_id`

Reply:
I think we can remove `stop_words` from the generated text in `stream_infer` and `infer` when `status.value` is `TRITON_STREAM_END`